tests/test_request_form_file.py

```diff
@@ -0,0 +1,177 @@
+from fastapi import FastAPI, File, Form
+from fastapi.testclient import TestClient
+
+app = FastAPI()
+
+
+@app.post("/files/")
+async def create_file(token: str = Form(), file: bytes = File()):
+    return {
+        "file_size": len(file),
+        "token": token,
+    }
+
+
+client = TestClient(app)
+
+openapi_schema = {
+    "openapi": "3.0.2",
```

Comment:
```suggestion
    "openapi": "3.1.0",
```
Current version is 3.1.0